lib/qonfig/plugins/vault/loaders/vault.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+# @api private
+# @since 0.25.0
+class Qonfig::Loaders::Vault < Qonfig::Loaders::Basic
+  # @return [Binding]
+  #
+  # @api private
+  # @since 0.25.0
+  VAULT_EXPR_EVAL_SCOPE = BasicObject.new.__binding__.tap do |binding|
+    Object.new.method(:freeze).unbind.bind_call(binding.receiver)
```

Comment:
at this moment qonfig supports rubies since 2.4, but **bind_call** is a method added in 2.7 :( 

can we rewrite this with `unbind.bind(binding.receiver).call` to support older rubies? :)